test/functional/wallet_keypool_topup.py

```diff
@@ -25,7 +25,7 @@ class KeypoolRestoreTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
-        self.extra_args = [['-deprecatedrpc=accounts'], ['-deprecatedrpc=accounts', '-keypool=100', '-keypoolmin=20']]
+        self.extra_args = [[], ['-keypool=100', '-keypoolmin=20']]
```

Comment:
What does keypoolmin do?

Reply:
Looks like this was added in d34957e17e8c9740104533aaf4a896e93548c87e and it is fine to just remove.

Reply:
removed